search-app/docs/api.md

```diff
@@ -5405,3 +5412,152 @@ Example un-healthy response body:
   }
 }
 ```
+#### <a name="generic-association"></a> Associate concepts
+
+A concept of any type can be associated to another concept of any type using the association API. Note this type of association is functionally the same as associations of tools, services, and variables to collections but, uses a different API route and is applicable to all possible concept relationships. In such fashion, any concept can be associated to any other concept, including concepts of the same type i.e. associate a variable to another variable. Associations can also include an optional data payload the purpose of which is to describe the association itself. Associations which do not initially have association details, may have details added simply by providing association details as a subsequent revision. A concept cannot be associated to itself.
```

Comment:
some text should be given on payload overwriting existing values on updates.

Also, I don't know about the comma after "to collections but,".

Reply:
I do expand on that topic in the: "Update Association details" section, perhaps it should be in the main section or deserves it's own TOC entry?